addons/search-my-stuff/userscript.js

```diff
@@ -25,13 +25,15 @@ export default async function ({ addon, global, console, msg }) {
     // Ignore input
     // 1) in <input> or <textarea>
     // 2) with Ctrl/Alt/Meta keys pressed (probably browser shortcut keys)
-    // 3) that is not represented by one alphanumeric key code (probably special letters)
+    // 3) that is a space character (browser shortcut key for paging down)
+    // 4) that is not represented by one alphanumeric key code (probably special letters)
     if (
       e.target instanceof HTMLInputElement ||
       e.target instanceof HTMLTextAreaElement ||
       e.ctrlKey ||
       e.altKey ||
       e.metaKey ||
+      e.key === " " ||
```

Comment:
@lisa-wolfgang Is this just an optimization (to avoid the regex?)

Reply:
I didn't know that this Regex code matched the space character.